library/src/scripts/components/headers/VanillaHeader.tsx

```diff
@@ -94,6 +95,16 @@ export class VanillaHeader extends React.Component<IProps, IState> {
         const containerElement = this.props.container || document.getElementById("vanillaHeader")!;
         containerElement.classList.value = outerCssClasses;
 
+        const themeLogo = getLogo(LogoType.DESKTOP);
+        let logoData = dummyLogoData;
```

Comment:
Can we rename this delete the `dummyData` file and just put this inline as the default?